Replicator/Inserter.cc

```diff
@@ -220,4 +213,16 @@ namespace litecore { namespace repl {
         return C4SliceResult(body);
     }
 
+    C4Collection* Inserter::collection() {
+        if (_collection)
+            return _collection;
+        
+        auto c4db = _db->insertionDB().useLocked();
+        auto coll = c4db->getCollection(replicator()->collection(collectionIndex())->getSpec());
```

Comment:
Perhaps make the name clearer, insertionCollection(), to contrast with replicator()->collection(collectionIndex())